content/tutorials/spatial_transform_crs_2/index.Rmd

```diff
@@ -0,0 +1,495 @@
+---
+title: "Coordinate operations and maps in R"
+description: "This tutorial shows how to do coordinate transformations and conversions in R. It also demonstrates how spatial data can be mapped with ggplot2 and mapview."
+authors: [florisvdh, ambermertens]
+date: 2021-09-16
+categories: ["r", "gis"]
+tags: ["gis", "r", "maps"]
+csl: https://raw.githubusercontent.com/citation-style-language/styles/master/research-institute-for-nature-and-forest.csl
+link-citations: TRUE
+bibliography: bibliography.yaml
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: gfm
+---
+
+```{r include = FALSE}
+library(knitr)
+opts_chunk$set(
+  echo = TRUE,
+  warning = FALSE,
+  comment = ""
+)
+options(stringsAsFactors = FALSE)
+git_root <- rprojroot::find_root(rprojroot::is_git_root)
+tutorial_root <- file.path(git_root, "content/tutorials/spatial_transform_crs_2")
+dir.create(file.path(tutorial_root, "data_gisclub"), showWarnings = FALSE)
+gisclubdata_path <- file.path(tutorial_root, "data_gisclub")
+```
+
+## Introduction and setup
+
+This tutorial originated while preparing an R-demonstration during a GIS club at INBO on 16 September 2021.
+(Further material of the GIS club is available [here](https://drive.google.com/drive/folders/1s5HZ5VjnkTbCQSE1vfFyMJz6PVIl4mHW).)
+
+A straightforward approach for transforming spatial data is available in another [tutorial](../spatial_transform_crs/).
+The current tutorial tackles a few extra aspects.
+
+The tutorial assumes you have some pre-existing knowledge:
+
+-   Basic knowledge about coordinate reference systems (CRSs) and geodetic datums; see [another tutorial](../spatial_crs_coding/) and the references and links therein.
+-   Knowing how to read geospatial files with [`sf`](https://r-spatial.github.io/sf).
+There is another [tutorial](../spatial_standards_vector/) demonstrating some aspects.
+
+In this tutorial we will apply conversions (without datum shift) and transformations (with datum shift) to `sf` and `sfc` objects.
+In the background, these operations use the [PROJ](https://proj.org) library.
+
+Let's load the needed R packages and prepare a more appropriate theme for mapping with [`ggplot2`](https://ggplot2.tidyverse.org/):
+
+```{r message=FALSE}
+library(dplyr)
+library(sf)
+library(ggplot2)
+theme_set(theme_bw())
+theme_update(panel.grid = element_line(colour = "grey80"))
+library(mapview)
+```
+
+Set up input data if needed:
+
+```{r}
+if (!file.exists(file.path(gisclubdata_path, "gemeenten_belgie.shp"))) {
+  googledrive::drive_download(googledrive::as_id("1oi-XvO54beBKcoTfbjVd8QRz-0j3Upfx"), 
```

Comment:
Is this downloadable for everybody?

Reply:
Yes, I made it that way, from the zip file I got from Amber (it's in my drive).

However the reference to the GISclub material (GDrive folder URL in markdown text) is open only to INBO personnel. What should we do about that @amber-mertens: open it up, or add that that material is INBO-only stuff?

Reply:
Is there already an open location for tutorial files? In my opinion we can open up access to the files but I am not sure if the GISclub drive can be used for that - should I ask the organizers?

Reply:
Good questions! The tutorials repository itself normally just contains the source of the tutorials (markdown and R code, plus figures and potentially data-objects, preferrably text-only). That's why I tend to put binary / larger content on some external storage like GDrive, and download it in git-ignored places with R code, but I'm open to other ideas on that. (@damianooldoni @hansvancalster ?)

AFAIK there's no single location appointed to store external files used by the tutorials repository; in fact with GDrive it's not essential since access in GDrive can be set at folder and file levels as we like, and we can use direct URLs in the tutorials. For oversight it seems like a good idea to start some folder like `PRJ_tutorials`  and consistently store material there. However, it may invoke unneeded duplication, e.g. if GISclub material would be opened up anyway - and in the light of maintenance (making updates).

Whether this GISclub's drive material is open to the public (it currently isn't), is indeed for the organizers to consider, so yes please ask if it's not up to you to decide that.

Reply:
My gut feeling is that a separate `PRJ_tutorials` folder is not needed and the approach you used in this tutorial is fine by me. There are not so many tutorials which need large files and the ones that do download it from some location (GDrive, Zenodo, ...).

Reply:
Solved: the GISclub folder is public now. I'll update the zipfile link to the one in this folder.